consumer/tests/ckan/.env

```diff
@@ -0,0 +1,14 @@
+
+CKAN_SITE_ID=default-test
+CKAN_SITE_URL=http://ckan-test:5000
+
+CKAN_PORT=5000
+#
+# Email settings
+CKAN_SMTP_SERVER=smtp.corporateict.domain:25
+CKAN_SMTP_STARTTLS=True
+CKAN_SMTP_USER=user
+CKAN_SMTP_PASSWORD=pass
+CKAN_SMTP_MAIL_FROM=ckan@ckan-test
+
+DATASTORE_READONLY_PASSWORD=datastore
```

Comment:
:thinking:

Reply:
The CKAN setup is only used for integration testing